tasks/managed/extract-index-image/extract-index-image.yaml

```diff
@@ -35,13 +38,13 @@ spec:
 
         RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/extract-index-image-results.json"
 
-        jsonBuildInfo=$(jq -cr .jsonBuildInfo "$(params.inputDataFile)")
-
-        indexImage=$(echo "$jsonBuildInfo" | jq -cr .index_image)
-        echo -n "$indexImage" > "$(results.indexImage.path)"
+        indexImage="$(jq -cr '.components[].index_image' \
```

Comment:
Don't need -c and -r together

Reply:
No. I don't remember what was the reason the -c entered here. Probably copy+paste from somewhere.